src/server/ast/symbolTable.ts

```diff
@@ -160,7 +160,8 @@ export class SymbolTable
 			return symbol
 		else if (this.parentTable)
 			return this.parentTable.find(ident)
-		return
+
+		return undefined
```

Comment:
Don't love this.. we know `return` on its own returns undefined, and that was chosen to reduce the amount that could be typed wrong.. but, if you can provide us a compelling reason for this change, then sure